configure_dag.py

```diff
@@ -15,12 +15,15 @@
 import pandas as pd
 
 import utils
+import more_itertools as mit
+import numpy as np
+import psycopg2
 
 
 ### Main functions.
 
 @utils.log_start_and_finish
-def get_exposure_info(ra: float, dec: float) -> pd.DataFrame:
+def get_exposure_info(ra: float, dec: float, minexp=None, maxexp=None) -> pd.DataFrame:
```

Comment:
[nit pick] after adding additional function arguments, it would be good to update the Args section of the docstring